src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -148,7 +148,9 @@ def prepare_logs_argument(c):
         with self.argument_context(scope) as c:
             c.argument(
                 'artifact_path', options_list=[
-                    '--artifact-path', '--jar-path', '-p'], help='If provided, deploy pre-built artifact (jar or netcore zip), otherwise deploy current folder as tar.')
+                    '--artifact-path', '--jar-path'], help='If provided, deploy pre-built artifact (jar or netcore zip), otherwise need to assign source_path.')
+            c.argument(
+                'source_path', nargs='?', const='.', help="If provided, deploy current or designative folder as tar, otherwise need to assign artifact-path.")
```

Comment:
Maybe `Upload the specified source folder to build and deploy. Default to the current folder.`